skills/sap-cap-capire/references/cli-complete.md

```
@@ -0,0 +1,574 @@
+# CAP CLI - Complete Command Reference
+
+**Source**: https://cap.cloud.sap/docs/cli/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)